procedure requests/README.md

```diff
@@ -4,23 +4,25 @@
+|------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+| meta.profile                                   | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-ProcedureRequest-1"`                                             |
+| identifier\[0].system                          | `"https://PSSAdaptor/{{losingOdsCode}}"` - where the `{{losingOdsCode}}` is the ODS code of the losing practice                               |
+| identifier\[0].value                           | ` PlanStatement / id \[@root] `                                                                                                               |
+| id                                             | ` PlanStatement / id \[@root] `                                                                                                               |
+| status                                         | `PlanStatement / text` <sup>1</sup>                                                                                                           |
+| intent                                         | fixed value = `"plan"`                                                                                                                        |
+| authoredOn                                     | ` PlanStatement / availabilityTime \[@value] ` or ` EhrComposition / availabilityTime \[@value] ` or `EhrComposition / author / time \[@value]` |
+| occurrenceDateTime                             | ` PlanStatement / effectiveTime / center \[@value] `                                                                                          |
+| subject                                        | reference to the mapped [Patient](../patient/README.md)                                                                                       |
+| note                                           | ` PlanStatement / text `                                                                                                                      |
+| requester.agent                                | ` PlanStatement / participant `                                                                                                               |
+| code                                           | ` PlanStatement / code ` <sup>2</sup> as described in the XML > FHIR section of [Codeable Concept](../codeable%20concept/README.md)           |
+| context                                        | reference to the associated [Encounter](../encounters/README.md)                                                                              |
+
+1. When `PlanStatement / text` starts with `Status:` the value immediately following this will be mapped to either `active`, `cancelled` or completed.
```

Comment:
It seems that completed is missing single quotes...

Reply:
resolved